config/main.py

```diff
@@ -668,7 +765,9 @@ def _get_disabled_services_list(config_db):
     return disabled_services_list
 
 
-def _stop_services():
+def _
```

Comment:
typo ?

Reply:
taken care. this was due to mere from master.  updated main.py to master branch and add new mclag file